lxd/api_cluster.go

```diff
@@ -2039,6 +2041,16 @@ func updateClusterCertificate(ctx context.Context, d *Daemon, r *http.Request, r
 
 	// First node forwards request to all other cluster nodes
 	if r == nil || !isClusterNotification(r) {
+		var err error
+
+		revert.Add(func() {
+			if err == nil {
```

Comment:
You shouldn't need to check err here as revert func should only be called if there has been an error.

Reply:
You could use fmt to print the error as if err is ever nil you would still want the warning logged as it means there is a bug.